src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml

```diff
@@ -46,6 +49,20 @@
                 </Setter>
             </Style>
 
+            <DrawingBrush x:Key="CheckersBrush" Viewport="0,0,0.25,0.25" TileMode="Tile">
+                <DrawingBrush.Drawing>
+                    <DrawingGroup>
+                        <GeometryDrawing Brush="White">
```

Comment:
Shouldn't white depend on the current theme?

Reply:
forgot, thanks

Reply:
fixed